Description: fix for signal-exit >= 4
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2023-08-17

--- a/lib/index.js
+++ b/lib/index.js
@@ -6,7 +6,7 @@
 
 const fs = require('fs')
 const MurmurHash3 = require('imurmurhash')
-const onExit = require('signal-exit')
+const { onExit } = require('signal-exit')
 const path = require('path')
 const { promisify } = require('util')
 const activeFiles = {}
